web3/contract.py

```diff
@@ -937,25 +1054,25 @@ def buildTransaction(self, transaction=None):
         Build the transaction dictionary without sending
         """
         if transaction is None:
-            built_transaction = {}
+            build_transaction = {}
```

Comment:
I disagree that this wording is clearer, but if you like `build` a lot better, I am ok with keeping it. Here's why I like `built` better: `build` is a verb, but the variable is holding a thing (noun) -- the transaction that is being built.

Counter-point: the analogous calls, like `estimate_gas_transaction` sound more like verbs, too, so `build` is a bit more consistent with those. On that point, I'd argue that the best solution would be to change them all to nouns, but maybe it's less obvious what the other nouns should be, like: `gas_estimate_transaction`?

Takeaway: this is a semantic nitpick, and a bit murky, so you make the call.

Reply:
Actually I am not so sure about it myself as well, I will revert this change and rename estimate_gas_transactions.

Reply:
I think I will leave `estimate_gas_transaction` and `transact_transaction` be, focus on the main task for now. Doesn't seem to provide more values any way :)